internals/scripts/create-npm-package.ts

```diff
@@ -17,9 +15,11 @@ export function createNpmPackage(opts: Options = {}) {
 
   crateTemplateFolder(opts);
 
-  shell.exec(`npm pack`, { silent: true });
+  const archiveFilename = shell
+    .exec(`npm pack`, { silent: true })
+    .stdout.replace(/^\s+|\s+$/g, '');
```

Comment:
I am really trying to ignore such coding where its NOT obvious at first. This project needs to be maintained over time and anything not explicit makes it hard for me considering that I dont have much time.

Reply:
If I add an explanatory comment to this line, or rewrite it more clearly, will it be possible to reconcile the changes?

This code change is needed to support the forked package name change. This is a great template and I really want to help support it.